guide/private-key-management/single-user-schemes.md

```diff
@@ -0,0 +1,326 @@
+
+The automatic cloud backup scheme gives users reasonable security against theft, while providing low risk for self-inflicted losses. 
+
+{% include image.html
+   image = "/assets/images/guide/private-key-management/schemes/cloud-backup.jpg"
+   retina = "/assets/images/guide/private-key-management/schemes/cloud-backup@2x.jpg"
+   alt-text = "Automatic cloud backup illustration"
+   caption = "Private keys are encrypted and stored in the cloud."
+   width = 800
+   height = 400
+%}
+
+#### Pros 
+- Low onboarding friction
+- Low risk of self-inflicted loss due to the automatic nature and redundant backups
+
+#### Cons
+- Private key in cloud storage increases risk of malicious third party access
+- User needs to trust their OS and cloud storage provider
+- Unless the backup includes the recovery phrase users will not be able to export their wallet to another product
+
+### How it works
+The wallet-application will generate a wallet with a keypair, then encrypt and back the key up to a location that the user is unlikely to lose access to, while hard for a malicious third party to gain access to. 
+
+With most implementations so far, the location will be the keychain or key-value store on a mobile device that is also backed up by the OS provider such as iOS (iCloud) or Android (Google Drive). 
+
+This makes the backup accessible by the user on a new device, should they lose the original, but only accessible by someone that can log into the user’s Apple or Google account. 
+
+### Best practice
+
+**When to use** 
+- When risk of self-inflicted loss is higher than loss from theft
+- For beginners, or products with progressive security
+- For lower values
+
+**When not to use** 
+- When storing high values
+- When risk of loss from theft is higher than self-inflicted loss
+- When complete self-sovereignty is important to the target audience
+
+**Variations** 
+- Storage location (keychain, iCloud, Google Drive, One Password etc.)
+- Additional user authentication (biometrics, pin)
+
+**Products that use this scheme** 
+- [Photon SDK]({{ 'https://photonsdk.org'}}) - for use by other wallets, in development
+- [Casa]({{ 'https://keys.casa'}})
+- [Pine]({{ https://pine.pm }}) - Lightning, beta
+- [Paymobil]({{ 'https://www.paymobil.co'}}) - not a bitcoin wallet
+- Bitcoin.com mobile wallet
+
+***
+
+## Manual backup / Recovery-phrase
+
+Manual backup of the recovery phrase have been the most common private key management scheme by far, since its proposal with [BIP39]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki'}}) in 2013. If you have used any non-custodial bitcoin product you are likely to have experienced the onboarding requirements of manual backups. 
+
+After creating a new wallet, you will be asked to manually backup a 12 (or 24 in rare instances) word recovery-phrase to a *safe place*. As the next step it will often ask you to verify that you did save it by having you input the phrase in the correct order. 
```

Comment:
>  12 (or 24 in rare instances) 

Do we have any data on this? How is it a rare instance? Personally I encountered 24 far more often. Maybe just say 12 or 24.

Reply:
Changing